### PR TITLE
If conversion should not convert PHI nodes (#77886)

### PR DESCRIPTION
**Pull request by a74nh**

### CODE REVIEW

**Comment by a74nh** [comment]
@jakobbotsch

**Comment by jakobbotsch** [comment]
Can you add the regression test? (Under src/tests/JIT/Regression/JitBlue/Runtime_77886)

**Comment by a74nh** [comment]
> Can you add the regression test? (Under src/tests/JIT/Regression/JitBlue/Runtime_77886)

What should I do about the Unhandled exception the test generates?  Running the test will give an "END EXECUTION - FAILED"

**Comment by jakobbotsch** [comment]
> What should I do about the Unhandled exception the test generates? Running the test will give an "END EXECUTION - FAILED"

You can either add `try { } catch (NullReferenceException)` at the top level or initialize `s_3` to avoid the exception.

File: src/tests/JIT/Regression/JitBlue/Runtime_77886/Runtime_77886.csproj [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
```
Comment (jakobbotsch) [inline src/tests/JIT/Regression/JitBlue/Runtime_77886/Runtime_77886.csproj:9]:
```suggestion
```

Unnecessary, we generally run tests both with and without tiered compilation in CI.

File: src/tests/JIT/Regression/JitBlue/Runtime_77886/Runtime_77886.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Generated by Fuzzlyn v1.5 on 2022-11-04 01:59:14
```
Comment (jakobbotsch) [inline src/tests/JIT/Regression/JitBlue/Runtime_77886/Runtime_77886.cs:1]:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

// Generated by Fuzzlyn v1.5 on 2022-11-04 01:59:14
```

**Comment by a74nh** [comment]
Oh, interesting - by accepting the review changes, it has not restarted the ci.

**Review by jakobbotsch — APPROVED** [review]

**Pull request merged**